packages/account/src/Sections/Verification/ProofOfOwnership/__test__/proof-of-ownership.spec.js

```diff
@@ -2,79 +2,85 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { ProofOfOwnership } from '../proof-of-ownership.jsx';
 import test_data from './test-data';
+import { StoreProvider, mockStore } from '@deriv/stores';
 
 describe('proof-of-ownership.jsx', () => {
     let ownership_temp;
     beforeAll(() => {
         ownership_temp = test_data;
     });
+    const store = mockStore();
     it('should render no poo required status page', () => {
+        store.client.account_status = {
```

Comment:
```suggestion
        const mock = mockStore({
            client: {
                account_status: {
                    authentication: {
                        ownership: { requests: [], status: 'none' },
                    },
                },
            },
        });

        const wrapper = ({ children }) => <StoreProvider store={mock}>{children}</StoreProvider>;

        render(<ProofOfOwnership />, { wrapper });
```
@yaswanth-deriv Please use `mockStore` properly in each test case 🙏🏻 Same in other test files 🙇🏻